src/kubernetes_api_objects/exec/label_selector.rs

```diff
@@ -69,7 +70,9 @@ impl LabelSelector {
     // TODO: prove it and maybe move to a different lib
     #[verifier(external_body)]
     pub fn matches(&self, labels: StringMap) -> (res: bool)
-        ensures res == self@.matches(labels@)
+        ensures
+            res == self@.matches(labels@),
+            res ==> self@.match_labels.is_None(),
```

Comment:
I think the second postcondition is wrong. "Matches" does not mean the labels is None

Reply:
You should just remove this postcondition